src/edfi_amt_data_lake/parquet/amt/base/academic_time_period_dim/main.py

```diff
@@ -0,0 +1,176 @@
+        metaPrefix=None,
+        recordPrefix=None,
+        errors="ignore"
+    )
+
+    session_with_term_descriptor_merged = pdMerge(
+        left=session_normalized,
+        right=term_descriptor_normalized,
+        how="inner",
+        leftOn=["termDescriptor"],
+        rightOn=["codeValue"],
+        suffixLeft="_session",
+        suffixRight="_term_descriptor_normalized"
+    )
+
+    session_with_term_descriptor_merged_and_grading_period_descriptor_normalized = pdMerge(
+        left=session_with_term_descriptor_merged,
+        right=grading_period_descriptor_normalized,
+        how="inner",
+        leftOn=["session_gradingPeriodReference.gradingPeriodDescriptor"],
+        rightOn=["codeValue"],
+        suffixLeft="_session_with_term_descriptor_merged",
+        suffixRight="_grading_period_descriptor_normalized"
+    )
+
+    result_data_frame = pdMerge(
+        left=session_with_term_descriptor_merged_and_grading_period_descriptor_normalized,
+        right=schoolYearTypes_normalized,
+        how="inner",
+        leftOn=["session_gradingPeriodReference.schoolYear"],
+        rightOn=["schoolYear"],
+        suffixLeft="_session_with_term_descriptor_merged_and_grading_period_descriptor_normalized",
+        suffixRight="_schoolYearTypes_normalized"
+    )
+
+    addColumnIfNotExists(result_data_frame, "LastModifiedDate")
+
+    result_data_frame["AcademicTimePeriodKey"] = (
+        result_data_frame["session_gradingPeriodReference.schoolId"].astype(str)
+        + "-" + result_data_frame["session_gradingPeriodReference.schoolYear"].astype(str)
+        + "-" + result_data_frame["termDescriptorId"].astype(str)
+        + "-" + result_data_frame["gradingPeriodDescriptorId"].astype(str)
+        + "-" + result_data_frame["beginDate"].astype(str).str.replace("-", "")
+    )
+
+    result_data_frame["SessionKey"] = (
+        result_data_frame["session_gradingPeriodReference.schoolId"].astype(str)
+        + "-" + result_data_frame["session_gradingPeriodReference.schoolYear"].astype(str)
+        + "-" + result_data_frame["sessionName"].astype(str)
+    )
+
+    result_data_frame["GradingPeriodKey"] = (
+        result_data_frame["gradingPeriodDescriptorId"].astype(str)
+        + "-" + result_data_frame["session_gradingPeriodReference.schoolId"].astype(str)
+        + "-" + result_data_frame["beginDate"].astype(str).str.replace("-", "")
+    )
+
+    result_data_frame = renameColumns(result_data_frame, {
+        "session_gradingPeriodReference.schoolYear": "SchoolYear",
+        "schoolYearDescription": "SchoolYearName",
+        "currentSchoolYear": "IsCurrentSchoolYear",
+        "session_gradingPeriodReference.schoolId": "SchoolKey",
+        "sessionName": "SessionName",
+        "codeValue_session_with_term_descriptor_merged": "TermName",
+        "codeValue_grading_period_descriptor_normalized": "GradingPeriodName",
+    })
+
+    result_data_frame = result_data_frame[columns]
```

Comment:
I think this line is unnecessary, since in the `return` we are returning just the columns needed for the parquet.